### ISSUE TITLE
blank fields are removed

### ISSUE THREAD

**Issue by JLmike7** [mgalante/jquery.redirect#49]
Any fields that have a blank string as a value are removed.  This does not match $.ajax()

For example $.ajax() would send:
foo[bar]: beep
foo[test]:
foo[bla]: boop

but $.redirect() sends:
foo[bar]: beep
foo[bla]: boop

**Comment by mgalante** [comment]
Yes... this part of the code is the responsable of that:
```javascript  
      } else if (values[propName].length < 1) {
        delete values[propName];
    }
```
I think we could remove it.
Now im a little bit bussy. If you are hurried, just remove that lines, if not, i will publish a new version soon.

**Comment by felipsbreno** [comment]
Hey @mgalante,

I'm not found a PR to fix this problem, so I'm opened a new merge to fix.

**Comment by mgalante** [comment]
fixed in jquery redirect 1.2.0 (check the new flag)
